src/Components/WebAssembly/WebAssembly/test/Hosting/SatelliteResourcesLoaderTest.cs

```diff
@@ -16,7 +16,6 @@ public class SatelliteResourcesLoaderTest
     {
         [Theory]
         [InlineData("fr-FR", new[] { "fr-FR", "fr" })]
-        [InlineData("zh-CN", new[] { "zh-CN", "zh-Hans", "zh" })]
```

Comment:
Is the issue that Windows has a different opinion than Mac/Linux for this culture?

If so it seems reasonable just not to have this particular test case. Is there any other test case with nontrivial rules (not of the form `{ "X-Y-Z", "X-Y", "X" }`)?

Reply:
Hmm, doesn't look like there are any non-trivial cases in Linux (haven't tested OSX). I wonder if the Linux implementation just uses the trivial fallback behavior.